app/controllers/fae/utilities_controller.rb

```diff
@@ -61,6 +70,32 @@ def can_toggle(klass, attribute)
       true
     end
 
+    def translate_request(language, en_text)
+      subscription_key = ENV['TRANSLATOR_TEXT_SUBSCRIPTION_KEY']
+      endpoint = ENV['TRANSLATOR_TEXT_ENDPOINT']
+      path = '/translate?api-version=3.0'
```

Comment:
Since it seems like this is built on the assumption that a specific service provider will be used, i don't think we need to hide the endpoint in an ENV var, one less thing to chase around as an ENV var.

Reply:
updated!